### ISSUE TITLE
Add documentations on accumulated batch normalization

### ISSUE THREAD

**Issue by andreped** [andreped/GradientAccumulator#58]
We could make something similar to what we already have to gradient accumulation [here](https://github.com/andreped/GradientAccumulator#what).

Although it should lie in the documentations and not the README.
